lib/Doctrine/Import/Builder.php

```diff
@@ -179,6 +179,14 @@ class Doctrine_Import_Builder extends Doctrine_Builder
      */
     protected $_phpDocEmail = '##EMAIL##';
 
+
+    /**
+     * Contains the actAs columns after running buildSetUp
+     *
+     * @var array
+     */
+    private $_actAsColumns = array();
```

Comment:
As we can use 'private', we should not prefix the property with _ : IIRC this was used in old php codes as a demarcation for private properties and methods

Reply:
The codebase looks like not following a clear rule so far, approx. half are with _ and the other half without. There is at least one example of having it mixed in the same class (https://github.com/FriendsOfSymfony1/doctrine1/blob/master/lib/Doctrine/Locking/Manager/Pessimistic.php#L44-L49) - I'm fine with both ways and changed it in the pr. If we define it that way, we could check upcoming prs to follow that rule.

Reply:
Yeah, I know! I've been working on another (bigger) OSS library where underscores are widely present, as they're there from the initial PHP5 implementation.
Writing good code now, and fixing what we can is a good habit IMO. Let's tackle the whole cleanup, at least for the "private" ones in another PR :)